NitroxModel/Packets/FabricatorEndCrafting.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using NitroxModel.DataStructures;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class FabricatorEndCrafting : Packet
```

Comment:
Should this also be renamed to Crafter?

Reply:
Yes, you are right, this would be coherent.